src/cpp/session/modules/SessionHelp.R

```diff
@@ -760,3 +760,42 @@ options(help_type = "html")
    fmt <- "help/doc/html/Search?pattern=%s&title=1&keyword=1&alias=1"
    sprintf(fmt, utils::URLencode(query, reserved = TRUE))
 })
+
+.rs.addFunction("followHelpTopic", function(topic, package)
+{
+   tryCatch({
+      # useful for topic like ">=" that gets encoded by the help 
+      # system as ">%3D"
+      topic <- utils::URLdecode(topic)
+
+      # first look in the specified package
+      file <- utils::help(topic, package = (package), help_type = "text", try.all.packages = FALSE)
```

Comment:
Curious why the parentheses around `(package)` are used?

Reply:
otherwise nse kicks in, borrowed from base R